test/demo.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
+source /WORK_REPO/venv/bin/activate
```

Comment:
This should be fine if user already did it before `sh ./demo.sh`, right? like re-activate, plz double-check.

Reply:
If I run `./demo.sh`, it will be fine whether the user already did it or not.
If I run `sh ./demo.sh`, it won't work whether the user already did it or not.